spec/components/candidate_interface/carry_over_interstitial_component_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+RSpec.describe CandidateInterface::CarryOverInterstitialComponent do
+  it 'renders the component' do
+    application_form = build(:completed_application_form)
+    result = render_inline(described_class.new(application_form: application_form))
+
+    expect(result.text).to include('Continue your application')
+  end
+
+  context 'after the new recruitment cycle begins' do
+    around do |example|
+      Timecop.freeze(CycleTimetable.apply_reopens(2022) + 1.day) do
+        example.run
+      end
+    end
+
+    it 'renders the correct academic years' do
+      application_form = build(:completed_application_form, recruitment_cycle_year: RecruitmentCycle.previous_year)
```

Comment:
Would be good to hardcode the year for the test — we'll have to fix later otherwise?